check_sites.py

```diff
@@ -25,20 +25,18 @@
 def parse_rws_json(rws_json_string, strict_formatting):
     """Attempts to parse `rws_json_string` as JSON and validate formatting if `strict_formatting` is true.
 
-    Returns a tuple of the JSON dict and None if there were no errors,
-    or None and the error message if there was an error.
+    Returns a tuple of the JSON dict and None if there were no formatting
+    errors, or None and the error message if there was an error.  If the file
+    is not proper json, `json.loads()` will throw a JSONDecodeError.
```

Comment:
Nit: Python docs typically say "raise" instead of "throw", since that's what the Python keyword is.

```suggestion
    is not proper json, `json.loads()` will raise a JSONDecodeError.
```